tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -418,6 +422,10 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
             StatsComputationEnabledInternal = config
                                      .WithKeys(ConfigurationKeys.StatsComputationEnabled)
                                      .AsBool(defaultValue: (IsRunningInGCPFunctions || IsRunningInAzureFunctionsConsumptionPlan));
+            if (ExperimentalAppsecStandaloneEnabledInternal)
+            {
+                StatsComputationEnabledInternal = false;
```

Comment:
> We may want to manually send a configuration telemetry entry here that shows that the setting was overridden

I agree

> Instead of this if, maybe you can add ExperimentalAppsecStandaloneEnabledInternal to the conditions in AsBool() above

Unfortunately that doesn't _quite_ work, because this wants to _force_ overriding the value, even if the customer explicitly enabled

> the original value was overridden

I think this is the correct option, because we're (potentially) superseding the customer's choice. We _could_ combine all these, and only _force_ (and record) the value _if_ it's true 🤔 Maybe that's the best option? e.g.

```suggestion
            if (ExperimentalAppsecStandaloneEnabledInternal && StatsComputationEnabledInternal)
            {
                telemetry.Record(ConfigurationKeys.StatsComputationEnabled, false, ConfigurationOrigins.Calculated);
                StatsComputationEnabledInternal = false;
```

Reply:
I didn't know about that kind of telemetry.
That's seems to be a good option like that: applied in f0ed01c6c0f0bff64f065096b85e7d75fd51f08c 😄